bricks/containers/src/main-view/index.tsx

```diff
@@ -95,6 +96,19 @@ class EoMainView extends ReactNextElement {
   @property({ type: Boolean })
   accessor bannerSunk: boolean | undefined;
 
+  /**
+   * 是否展示 banner（包括页面标题，面包屑，工作栏）
+   * @default true
+   */
+  @property({ type: Boolean })
+  accessor showBanner: boolean | undefined;
+
+  /**
+   * 是否没有边距
+   */
+  @property({ type: Boolean })
```

Comment:
忘了说，只影响 css 的属性，可以设置 `render: fasle`

```suggestion
  @property({ type: Boolean, render: true })
```

<img width="611" alt="image" src="https://github.com/easyops-cn/next-bricks/assets/2338946/bfdcc536-86cf-4939-805f-81623af87983">